src/i18n/en/ui.ts

```diff
@@ -60,6 +60,9 @@ export default {
 	'fallbackContent.notice':
 		'This page is not yet available in your language, so we’re showing you the English version. You can help by translating it!',
 	'fallbackContent.linkText': 'Learn more about how you can contribute',
+	// Outdated content notice shown when a page is outdated
+	'outdatedContent.notice':
+		'The content on this page is behind the english translation & may be outdated. You can help by translating it!',
```

Comment:
```suggestion
		'The content on this page is an older translation that requires updating to match the current English version. You can help by translating it! Visit our [contributing](/en/contribute/) page for details.',
```
Maybe something like this?

At least:
- English needs to be capitalized
- the English is not a translation itself :)

Reply:
Yep, that sounds much better :sweat_smile:
Yeah, the plan would be to translate this, like the fallback notice has been